docs/rules/no-named-import.md

```diff
@@ -7,80 +7,81 @@ also creates consistency inside a codebase because some people use named imports
 
 ### Options
 
-This rule has two different properties that can be used (not at the same time): `allow` and `forbid`. Both are an array of strings. By default this rule allows nothing, so every named import will error.
```

Comment:
removing "allow" and "forbid" is a breaking change, and isn't something i'll find acceptable.